### PR TITLE
Insta generate

### PR DESCRIPTION
**Pull request by titusz**
Generate ISCC-CODE instantly

### CODE REVIEW

File: iscc_web/api/media.py [hunk 1]
```diff
@@ -45,71 +38,48 @@ async def upload_file(self, request: Request):
...
+        # Create package directory
+        media_id, package_dir = await self.create_package()
 
-        # Save Uploaded data
-        await self.write_meta(media_id, file_meta)
-        async with aiofiles.open(self.file_path(media_id), "ab") as outf:
-            async for chunk in request.stream():
-                await outf.write(chunk)
+        # Store file metadata
+        ip = request.client_ip
```
Comment (strayer) [inline iscc_web/api/media.py:45]: I want to note that an IP address is considered as personal data by GDPR, so we will most likely need to add some kind of consent to or before the upload of files if it stores the IP address.

File: pyproject.toml [hunk 2]
```diff
@@ -23,17 +26,29 @@ black = "^22.8"
...
 
 [tool.poe.tasks]
+formatyaml = { cmd = "poetry run python -m develop.format_yaml", help = "Reformat yaml files"}
+codegen = { cmd = "datamodel-codegen", help="Build code from OpenAPI Schema"}
 format = { cmd = "poetry run black .", help = "Code style formating with black" }
+lf = { cmd = "poetry run python -m develop.lf", help = "Convert line endings to LF" }
 test = { cmd = "poetry run pytest", help = "Run tests" }
 run = { cmd = "uvicorn iscc_web.asgi:application --reload", help = "Run dev server"}
-all = ["format", "test"]
+all = ["formatyaml", "codegen", "format", "lf", "test"]
```
Comment (strayer) [inline pyproject.toml]: Not sure if I like that the `all` taks is blindly formatting all yaml files…

File: iscc_web/api/media.py (hunk 1 above)
Comment (strayer) [inline iscc_web/api/media.py]: I want to note that an IP address is considered personal data by GDPR, so we will most likely need to add some kind of consent to or before the upload of files if it stores the IP address.

File: pyproject.toml (hunk 2 above)
Comment (titusz) [reply pyproject.toml]: Changed to only format openapi.yaml

File: iscc_web/api/media.py (hunk 1 above)
Comment (titusz) [reply iscc_web/api/media.py]: Changed to not store client ip.

**Pull request merged**